### DISCUSSION TITLE
"Properties" isn't a good name for the button to configure ExplorerPatcher

### DISCUSSION THREAD

**Discussion by skeddles** [valinet/ExplorerPatcher#670] [Ideas]
I looked all over and couldn't figure out how to get into the settings for this program. I had to look it up. 
"Properties" is not what I'd expect settings to be under. I'd expect the "taskbar settings" button to open your settings menu, though I understand not wanting to change that in case people want to access the windows settings section.

But I think the link for ExplorerPatcher should at least be made more intuitive and clear, for example something like "ExplorerPatcher Settings".  

Would be nice if the settings window's title reflected that as well, rather than just "properties".

**Comment by valinet** [comment]
A custom label means one string that has to be translated in all the languages supported by Windows, otherwise it would add an English-only string to a menu that is potentially localized and would look rather bad. So no, it won't be implemented. 

Also, once you found it, the name doesn't matter at all. Furthermore, you do not have to look all over. You just had to spend 2 minutes to read the README file, which is clearly called "README" for a reason, and there it tells you how to get to the program's settings. So yeah, I am sorry, but your approach lead to you wasting your time, not the supposed unintuitiveness of that label.

So yeah, it won't be changed. "Properties" is fine, non-intrusive and works okay once you know it is located there.
